scripts/bump.sh

```diff
@@ -59,29 +60,9 @@ echo "::set-output name=version::$NEXT_EXTENSION_VERSION"
 jq ".version = \"$NEXT_EXTENSION_VERSION\" | \
     .prisma.version = \"$SHA\" | \
     .dependencies[\"@prisma/get-platform\"] = \"$NPM_VERSION\"" \
-    ./../language-server/package.json > ./../language-server/package.json.bk
-
-mv ./../language-server/package.json.bk ./../language-server/package.json
-mv ./package.json.bk ./package.json
-
-npm install
-
-(
-cd ./../language-server
-npm install
-)
+./packages/language-server/package.json >./packages/language-server/package.json.bk
 
-(
-cd ./../..
-npm install
-)
+mv ./packages/language-server/package.json.bk ./packages/language-server/package.json
+mv ./packages/vscode/package.json.bk ./packages/vscode/package.json
 
 echo "Bumped prisma.version in package.json from $OLD_SHA to $SHA"
```

Comment:
Line 17 path is incorrect and causes 
![image](https://user-images.githubusercontent.com/746482/85740362-32100000-b71f-11ea-95f2-3c924751eb97.png)